study.md

```diff
@@ -112,13 +124,16 @@ From a collection of galaxies, update the `name` attribute of the record with
 the `designation` attribute of "NGC 224" to "Andromeda".
 
 ```ruby
-# your answer here
+galaxies = galaxy.find_by(name:'NGC')
+gala
 ```
 
 ## Delete
 
 From a collection of characters, delete the record with the `id` attribute of 4.
 
 ```ruby
-# your answer here
+charcter = Character.find_by(id: 4)
+character.destroy
```

Comment:
The typo on the previous line will cause this line to throw an error since `character` doesn't exist.